### DISCUSSION TITLE
gesp-1-bcqm3020/

### DISCUSSION THREAD

**Discussion by giscus** [lihongzheshuai/lihongzheshuai.github.io#348] [General]
# gesp-1-bcqm3020/

稍微增加了一点点难度，输入后，根据题目要求进行计算，然后按要求输出结果。即跟之前的题(BCQM3017-3018)比起来多了计算环节，计算就小学应用题水平。

https://www.coderli.com/gesp-1-bcqm3020/

**Comment by Niuger** [comment]
2024.9.28打卡
